common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilter.java

```diff
@@ -81,6 +81,11 @@ int getVersionNumber() {
   public abstract long bitSize();
 
   /**
+   * Swamidass & Baldi (2007) approximation for number of items in a Bloom filter
```

Comment:
Please describe the method first and its properties (approximation error).  Then put the reference in `@seealso` with a permanent link to the paper: https://dx.doi.org/10.1021%2Fci600526a